### ISSUE TITLE
Exception on address info

### ISSUE THREAD

**Issue by cabeljunky** [ritiek/trunofficial#9]
When trying to get the  "31203038502" or "31203038502" the script fails to retrieve the data.
But if you use the URL it will return information without the area and city. 
This will result in an exception. A solution can be to add the following:

trunofficial/trunofficial.py (version 0.1.6)

```
class _Address:
    def __init__(self, address):
        try:
            self.area = address['area']
            self.city = address['city']
        except KeyError:
            self.area = None
            self.city = None
        self.countrycode = address['countryCode']
        self.timezone = address['timeZone']
        self.type = address['type']

```

**Comment by N3V0N** [comment]
could you send a link to the code you are using ?

**Comment by ritiek** [comment]
Thanks for reporting this. I'd love it if you could send a pull request to fix the same. :smile:

**Comment by ritiek** [comment]
Closed by #11.
